### PR TITLE
added about us page

### PR DESCRIPTION
**Pull request by khushi935**
## Related Issue
  -About us page added
Closes:    #8 
#### Describe the changes you've made
I created aboutus.js, aboutus.css, images folder in src->components, edit app.js, navbar.js, and footer.js.
#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

## Checklist:
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 | <b>updated screenshot </b> |
 Screenshot (82)
Screenshot (81)
Screenshot (80)

### CODE REVIEW

**Review by pankajkumarbij — APPROVED** [review]

File: src/components/Aboutus.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import "./Aboutus.css";
+import CardDeck from 'react-bootstrap/CardDeck';
+import Card from 'react-bootstrap/Card';
+import image from "../images/why-us.png";
+import mission from "../images/mission.png";
```
Comment (luckykumarirai) [inline src/components/Aboutus.js:6]: All images should be in the public folder.so put all images in public and change the path.

**Review by luckykumarirai — CHANGES_REQUESTED** [review]

File: src/components/Aboutus.js (hunk 1 above)
Comment (khushi935) [reply src/components/Aboutus.js:6]: Screenshot (83)
I am getting an error. Please have a look.

Comment (luckykumarirai) [reply src/components/Aboutus.js:6]: > Screenshot (83)
> I am getting an error. Please have a look.

No need to maintain a ```./../public/xyz.png```  path you can only write the direct name of images like ```src="images/xyz.png"```.
Try with this.

Comment (khushi935) [reply src/components/Aboutus.js]: I did this, now it's not showing an error. Please have a look.

**Review by luckykumarirai — APPROVED** [review]

**Pull request merged**